types/aliases.go

```diff
@@ -193,6 +193,7 @@ var (
 	FixtureCheckRequest       = v2.FixtureCheckRequest
 	FixtureCheckConfig        = v2.FixtureCheckConfig
 	FixtureCheck              = v2.FixtureCheck
+	FixtureProxyCheck         = v2.FixtureProxyCheck
```

Comment:
FWIW it's not necessary to add these aliases - they were only added for compatibility and gradual code repair, as the things in core/v2 were moved there from the types package.